hyperspy/tests/axes/test_axes_manager.py

```diff
@@ -263,3 +263,28 @@ def test_setting_indices_coordinates():
     s.axes_manager.indices = (2, 2)
     assert s.axes_manager.indices == (2, 2)
     assert m.call_count == 6
+
+class TestAxesHotkeys:
+
+    def setup_method(self, method):
+        s = Signal1D(zeros(7 * (5,)))
```

Comment:
``np.zeros`` after importing numpy using ``import numpy as np``?

Reply:
Added import numpy at top of doc.